src/main/java/com/vackosar/gitflowincrementalbuild/boundary/UnchangedProjectsRemover.java

```diff
@@ -157,5 +217,39 @@ private void printDelimiter() {
     private boolean matchesAny(final String str, Collection<Pattern> patterns) {
         return patterns.stream().anyMatch(pattern -> pattern.matcher(str).matches());
     }
-}
 
+    private static class ProjectSelectionUtil {
+
+        static Set<MavenProject> gatherSelectedProjects(MavenSession mavenSession) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/gitflow-incremental-builder/gitflow-incremental-builder/pullRequest?prid=5271138)